components/TitleBar.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <div class="TitleBar">
+    <div class="Background-Wrapper">
+      <div class="header-container">
+        <h2>
+          <span class="headertitle">{{ header }}</span>
+        </h2>
+        <p>
+          <span class="headersubtitle">{{ subheader }} </span>
+        </p>
+      </div>
+      <v-card class="card">
+        <h4>
+          <span class="cardheader">
+            Help us raise Bay Area COVID awareness.
+          </span>
+        </h4>
+        <p>
+          <span class="cardsubheader">
+            Have ways to improve this site? We welcome all who wish to help
+            contribute!
+          </span>
+        </p>
+        <a href="/about">
+          <v-card color="#4738AC" class="learn">
+            <h5>
+              <span class="learnheader">Learn more about the Project</span>
+            </h5>
+          </v-card>
+        </a>
+      </v-card>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+export default {
+  name: 'TitleBar',
+  props: {
+    header: {
+      type: String,
+      default: ''
+    },
+    titleId: {
+      type: String,
+      default: ''
+    },
+    subheader: {
+      type: String,
+      default: ''
+    }
+  }
+}
+</script>
+<style lang="scss">
+.TitleBar {
+  .Background-Wrapper {
+    background: url('../static/titlebar_bg.png') center no-repeat;
+    height: 340px;
+    position: relative;
+    width: 1440px;
+    height: 340px;
+    left: 0px;
+    right: 0px;
+    top: 0px;
```

Comment:
same comment as AboutTitleBar above.

Reply:
I did the same thing for AboutTitleBAr above